src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -414,6 +416,12 @@ blockCommands = M.fromList $
   , "pagebreak"
   ]
 
+blockTextcolor :: PandocMonad m => LP m Blocks
+blockTextcolor = do
+    skipopts 
+    color <- braced
+    divWith ("",[],[("style","color: " ++ color)]) <$> grouped block <* notFollowedBy inline
+
```

Comment:
@jgm, sorry for the delay. I added your suggestions, however, now test first test is failing. I'm not sure why pandoc does not recognise `\textcolor` as inline anymore.